lib/caesar.rb

```diff
@@ -1,6 +1,28 @@
+require './test/caesar_test'
+
 class Caesar
-  attr_reader :key
+
+  attr_accessor :string
+                :number
+                :char_map
+                :letters
 
   def initialize
+    @string = "the quick brown"
```

Comment:
Think about how you would structure this for any text that gets encoded, not just the quick brown... etc etc